frontend/src/domain/use_cases/map/updateMapInteractionListeners.ts

```diff
@@ -13,20 +13,20 @@ export enum MapInteractionListenerEnum {
 export const updateMapInteractionListeners = (listener: MapInteractionListenerEnum) => dispatch => {
   switch (listener) {
     case MapInteractionListenerEnum.DRAW_ZONE_OR_POINT:
-      openDrawLayerModal(dispatch)
+      openDrawLayerModal(dispatch, true)
```

Comment:
question: est-il possible que, dans le futur on est d'autres items à afficher ou cacher ? Si oui, on risque de se retrouver avec une palanquée de true / false à devoir ajouter. 

suggestion: on pourrait utiliser un attribut option qui prend en propriété les mêmes termes que dans le setDisplayItems.
Ca donnerait un truc dans ce genre là : 

```suggestion
      openDrawLayerModal(dispatch, { displayLayersSidebar: true })
```
A mon sens, ca permet aussi une meilleure lisibilité sur ce que fait openDrawLayerModal. Qu'en penses-tu ?

Reply:
on a tendance a ne pas anticiper les choses dans le code pour ne pas complexifier inutilement les choses parce que "peut-être qu'on devra faire ça", ou "si plus tard on doit ajouter ça".
Si ca te va tant qu'on a seulement une propriété je laisse le booléen, et si on doit en ajouter on fera un attribut option.
Pour la lisibilité, la fonction étant juste en dessous je trouve que ça reste assez clair (toujours parce qu'il n'y a qu'un propriété pour le moment)